### PR TITLE
feat: expand & collapse native token container

### PR DESCRIPTION
**Pull request by impelcrypto**
**Pull Request Summary**

* feat: expand & collapse native token container
  * native ASTR token to be folded
  * First visitor to have the on-boarding message 
* fix: widen the max width (on the assets page) as same as dApp Staking page (1100px -> 1300px)
* fix: changed asset category name
    * XVM (Cross-Virtual Machine) Assets
    * Cross-chain (XCM) Assets 

**Check list**

- [ ] contains breaking changes
- [x] adds new feature
- [ ] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [x] tested on mobile devices

**Release notes:**

feat: expand & collapse native token container

**This pull request makes the following changes:**

**changes**

* feat: expand & collapse native token container
* fix: widen the max width (on the assets page) as same as dApp Staking page (1100px -> 1300px)
* fix: changed asset category name

=Before=

image

=After=

image

image

### CODE REVIEW

**Comment by bobo-k2** [comment]
Asset page doesn't show data for me on test env. I have spinner running and nothing happens.

**Comment by impelcrypto** [comment]
> Asset page doesn't show data for me on test env. I have spinner running and nothing happens.

@bobo-k2 @Kahonnohak  Fixed, please take a look again.

**Comment by Kahonnohak** [comment]
Thank you @impelcrypto few comments from me please take a look.

- The token amount and the buttons are not aligned at the moment, please take a look.
スクリーンショット 2023-01-27 14 18 35


- The pop-up for mobile: could you make it the same width as other containers please?
スクリーンショット 2023-01-27 14 38 45


- (transferable) doesn't look good and I think it's better to place (transferable) in front. Please check the design again. 
Native-XVM 1600 Update


- Need a bar in the title section: Please see dApp Staking page (I see now it's not actually a part of the title as a component)
スクリーンショット 2023-01-27 14 29 17

**Comment by impelcrypto** [comment]
@Kahonnohak Updated, please take a look

**Comment by Kahonnohak** [comment]
Thank you and lovely!!

I spotted 2 places.
- The close button on the light theme
- Could you do another effort..!?

スクリーンショット 2023-02-06 16 57 36


スクリーンショット 2023-02-06 16 53 57
スクリーンショット 2023-02-06 16 54 09

**Comment by Kahonnohak** [comment]
Amazing looks clean!

I noticed that the mobile ASTR section is different from the design which is okay with me but just two things,

1) Padding looks too tight so please align it with ASTR ticker
2) Give a 6px radius to the corners

スクリーンショット 2023-02-07 8 19 32


Thanks!!

**Comment by impelcrypto** [comment]
@Kahonnohak  Fixed, please take a look.
image

File: src/hooks/useAccount.ts [hunk 1]
```diff
@@ -4,6 +4,8 @@ import { useStore } from 'src/store';
 import { computed, ref, watch } from 'vue';
 import { SubstrateAccount } from 'src/store/general/state';
 
+export const EthereumExtension = 'Ethereum Extension';
```
Comment (sirius651) [inline src/hooks/useAccount.ts:7]: Just tiny thing but I'm confused a bit it's variable. maybe if it's constant value, uppercase might be better?

**Review by sirius651 — APPROVED** [review]

File: src/hooks/useAccount.ts (hunk 1 above)
Comment (impelcrypto) [reply src/hooks/useAccount.ts]: @sirius651 Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/674/commits/24e171d94a7ef9e40cbbd0ec92424af81bcc4301) commit.

**Pull request merged**